src/Ouroboros/Network/ConsumersAndProducers.hs

```diff
@@ -1,14 +1,18 @@
 {-# LANGUAGE BangPatterns        #-}
+{-# LANGUAGE DataKinds           #-}
+{-# LANGUAGE KindSignatures      #-}
 {-# LANGUAGE NamedFieldPuns      #-}
 {-# LANGUAGE RecordWildCards     #-}
 {-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications    #-}
```

Comment:
Why are we adding unused language pragmas?

Reply:
Clearly I haven't removed everything I intended to.